packages/tocco-theme/src/ToccoTheme/DefaultTheme.js

```diff
@@ -1,141 +1,66 @@
-/* COLORS
- * base.paper and base.text are the very basic colors
- * base.fill and primary.fill are used for backgrounds which change by state
- * base.line and primary.line are use for text and borders which change by state
- * line and fill must have a high contrast
- * primary.fill and pirmary.line must have the same hue but differentiate in saturation
- * and/or brightness to align visual perception
- * primary line must have a high contrast to to base fill
- * primary fillContrast could be set if the contrast between primary fill and base line is not sufficient
- * color[0] is used as default (stateless)
- * color[1] is used for hover state
- * color[2] is used for active state
-*/
-
-const colors = {
-  base: {
-    fill: [
-      '#EEEEEE', // Grey 200
-      '#E0E0E0', // Grey 300
-      '#BDBDBD' // Grey 400
-    ],
-    line: [
-      '#424242', // Grey 800
-      '#303030', // Grey 850
-      '#212121' // Grey 900
-    ],
-    paper: '#FFFFFF',
-    text: '#212121' // Grey 900
+export default {
```

Comment:
file should be called defaultTheme i guess. only react components are pascal-case